lib/deprecated/pure/sockets.nim

```diff
@@ -118,6 +118,7 @@ type
     IPPROTO_IPV6,       ## Internet Protocol Version 6. Unsupported on Windows.
     IPPROTO_RAW,        ## Raw IP Packets Protocol. Unsupported on Windows.
     IPPROTO_ICMP        ## Control message protocol. Unsupported on Windows.
+    IPPROTO_ICMPV6      ## Control message protocol for IPv6. Unsupported on Windows.
```

Comment:
As @dom96 already pointed out, adding this makes `case` at line 232 not cover all the cases.

But my question is: should this file be modified at all, given that at the top of it there is:
```
**Warning:** Since version 0.10.2 this module is deprecated.
```

(version 0.10.2 was released 4 years ago)

Reply:
I was wondering the same. Removed.